docs/ADRs/0005-python-tooling.md

```diff
@@ -0,0 +1,44 @@
+---
+    title: 0005 Python coding standard and tools
+    adr:
+        author: Berry den Hartog
+        created: 28-Feb-2024
+        status:  accepted
+---
+
+## Context
+
+In modern software development, maintaining code quality is crucial for readability, maintainability, and collaboration.
+Python, being a dynamically typed language, requires robust tooling to ensure code consistency and type safety. Manual
+enforcement of coding standards is time-consuming and error-prone. Hence, adopting automated tooling to streamline this
+process is imperative.
+
+## Decision
+
+We will use these standards and tools for our own projects:
+
+- [google style guide](https://google.github.io/styleguide/pyguide.html)
```

Comment:
this is a long read, are we expected to read and follow it, or is this a guideline enforced by tooling mentioned below?

Reply:
I think it is worth it to read it though

Reply:
Some of the rules are indeed enforced by the tooling. but not all can be enforced by tooling. 

I would propose that the style guide is more a guide then a hard rule. It helps you have discussions during reviews about the code and helps you guide when writing code. i do not except anyone to know it by heart.

Reply:
Ok!